### PR TITLE
clockwork-cli: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/clockwork-cli.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# clockwork-cli
+
+> A command line interface for the Clockwork PHP debugging framework.
+
+- Monitor Clockwork logs for the current project:
```
Comment (sbrl) [inline pages/common/clockwork-cli.md]: Monitor them for what?

Comment (pxgamer) [reply pages/common/clockwork-cli.md]: Monitor them for requests, it basically allows you to debug HTTP requests, etc. for Laravel.

Comment (sbrl) [reply pages/common/clockwork-cli.md]: Ah, cool! I see.

(And fellow Ubuntu user! Hi!)

Comment (pxgamer) [reply pages/common/clockwork-cli.md]: Not my screenshot, but I do use Ubuntu 18.10. :stuck_out_tongue:

**Comment by pxgamer** [comment]
Ping @agnivade, @mebeim

File: pages/common/clockwork-cli.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`clockwork-cli`
+
+- Monitor Clockwork logs for a specific project:
+
+`clockwork-cli {{path/to/directory}}`
+
+- Monitor Clockwork logs for multiple projects:
+
+`clockwork-cli {{path/to/directory_1}} {{path/to/directory_2}}`
```
Comment (mebeim) [inline pages/common/clockwork-cli.md]:
```suggestion
`clockwork-cli {{path/to/directory1 path/to/directory2 …}}`
```

As per #2745, I would suggest this change. Only a consistency suggestion though, the example looks clear to me either way.

Comment (pxgamer) [reply pages/common/clockwork-cli.md]: I've just noticed one thing with using the unicode character, it doesn't seem to render as underlined text on the node client.
image
image

Comment (mebeim) [reply pages/common/clockwork-cli.md]: @pxgamer oh, damn... that could be a limitation of ASCII escape codes? Perhaps we could use the triple dot to improve compatibility or make the client translate it. IDK. That's a separate issue though, should be proposed as an issue and talked about.

**Comment by mebeim** [comment]
Good to go, just waiting for the build fix in #2750 to be merged so this can be merged too.

**Comment by sbrl** [comment]
Can I force-merge this, @mebeim?

**Comment by mebeim** [comment]
@sbrl either force-merge or @pxgamer should rebase on `master`. It is ok either way IMO :slightly_smiling_face:

File: pages/common/clockwork-cli.md (hunk 2 above)
Comment (sbrl) [reply pages/common/clockwork-cli.md]: Hrm. That's a bug in the client then.

**Pull request merged**